uni/lib/controller/local_storage/app_library_reservation.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:uni/model/entities/library_reservation.dart';
+
+import 'package:uni/controller/local_storage/app_database.dart';
+
+class LibraryReservationDatabase extends AppDatabase {
+  LibraryReservationDatabase()
+      : super('reservations.db', [
+          '''CREATE TABLE RESERVATION(
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        room TEXT,
+        startDate TEXT,
+        duration_hours INT,
+        duration_minutes INT
```

Comment:
`duration_hours` seems to be unused and probably should be removed and `duration_minutes` should be `duration` to match the key name of `LibraryReservation.toMap()`